code/game/machinery/fax_machine.dm

```diff
@@ -515,6 +515,85 @@ GLOBAL_LIST_EMPTY(alldepartments)
 	target_department = "Brig"
 	network = "USCM High Command Quantum Relay"
 
+/obj/structure/machinery/faxmachine/backpack
+	name = "\improper Portable Press Fax Machine"
+	desc = "A standard issue portable fax machine for civilian reporters. Functions off of an internal battery. Cannot receive faxes while being worn. It is currently deployed. Click-drag the device towards you to pick it up."
+	needs_power = FALSE
+	use_power = USE_POWER_NONE
+	icon_state = "fax_backpack"
+	health = 150
+
+/obj/item/device/fax_backpack
+	name = "\improper Portable Press Fax Machine"
+	desc = "A standard issue portable fax machine for civilian reporters. Functions off of an internal battery. Cannot receive faxes while being worn. It is currently undeployed. Activate the device inhand to deploy it."
+	w_class = SIZE_HUGE
+	flags_equip_slot = SLOT_BACK
+	flags_item = ITEM_OVERRIDE_NORTHFACE
+	icon = 'icons/obj/structures/machinery/library.dmi'
+	icon_state = "fax_backpack"
+	item_state = "fax_backpack"
+
+/obj/item/device/fax_backpack/attack_self(mob/user) //activate item version fax inhand to deploy
+	..()
+	if(!ishuman(user))
+		return
+	var/turf/T = get_turf(usr)
+	if(istype(T, /turf/open))
+		var/turf/open/floor = T
+		if(!floor.allow_construction)
+			to_chat(user, SPAN_WARNING("You cannot deploy \the [src] here, find a more secure surface!"))
+			return FALSE
+	var/fail = FALSE
+	if(T.density)
+		fail = TRUE
+	else
+		for(var/obj/X in T.contents - src)
+			if(X.density && !(X.flags_atom & ON_BORDER))
+				fail = TRUE
+				break
+			if(istype(X, /obj/structure/machinery/defenses))
+				fail = TRUE
+				break
+			else if(istype(X, /obj/structure/window))
+				fail = TRUE
+				break
+			else if(istype(X, /obj/structure/windoor_assembly))
+				fail = TRUE
+				break
+			else if(istype(X, /obj/structure/machinery/door))
+				fail = TRUE
+				break
+	if(fail)
+		to_chat(usr, SPAN_WARNING("You can't deploy \the [src] here, something is in the way."))
+		return
+	to_chat(user, "You begin to deploy \the [src]...")
+	if(do_after(user, 45, INTERRUPT_ALL|BEHAVIOR_IMMOBILE, BUSY_ICON_BUILD))
+		to_chat(user, SPAN_NOTICE("You deploy \the [src]."))
+		var/obj/structure/machinery/faxmachine/backpack/deployedfax = new(user.loc)
+		transfer_label_component(deployedfax)
+		qdel(src)
+		return
+	return ..()
```

Comment:
You're calling parent implementation both at the start of the proc tool. Pick one; not both.